layouts/partials/navbar.html

```diff
@@ -56,6 +56,12 @@
             {{ if $external }}target="_blank" rel="noreferer"{{ end }}
             class="hx-text-sm contrast-more:hx-text-gray-700 contrast-more:dark:hx-text-gray-100 hx-relative -hx-ml-2 hx-hidden hx-whitespace-nowrap hx-p-2 md:hx-inline-block {{ $activeClass }}"
           >
+            {{ with .Params.hero_icon -}}
+              <span style="float: left; padding-right: 0.3em; padding-top: 2px; opacity: 0.75">
```

Comment:
Can we use [Tailwind CSS](https://tailwindcss.com/docs/utility-first) classes here?

Reply:
I bet we could. I was hoping you'd know what they are. :-)

Reply:
I can begin converting them.